From f15b55a17f4285279d20d30ce2ad5d487ebd770e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alexander=20K=C3=B6plinger?= <alex.koeplinger@outlook.com>
Date: Mon, 5 Oct 2020 19:04:13 +0200
Subject: [PATCH] build-commons.sh: check for OpenSSL 1.1 on macOS (#43037)

We're installing openssl@1.1 from Homebrew so we should check that version.
It looks like CI Macs don't set the /usr/local/opt/openssl symlink anymore.

Upstream PR: https://github.com/dotnet/runtime/pull/43037
---
 eng/native/build-commons.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/native/build-commons.sh b/eng/native/build-commons.sh
index 2e94d68fe69..dc1641f5a49 100755
--- a/eng/native/build-commons.sh
+++ b/eng/native/build-commons.sh
@@ -55,7 +55,7 @@ check_prereqs()
         if ! pkg-config openssl ; then
             # We export the proper PKG_CONFIG_PATH where openssl was installed by Homebrew
             # It's important to _export_ it since build-commons.sh is sourced by other scripts such as build-native.sh
-            export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig
+            export PKG_CONFIG_PATH=/usr/local/opt/openssl@1.1/lib/pkgconfig
             # We try again with the PKG_CONFIG_PATH in place, if pkg-config still can't find OpenSSL, exit with an error, cmake won't find OpenSSL either
             pkg-config openssl || { echo >&2 "Please install openssl before running this script, see https://github.com/dotnet/runtime/blob/master/docs/workflow/requirements/macos-requirements.md"; exit 1; }
         fi
-- 
2.25.2

